_translations/nl.yml

```diff
@@ -182,6 +182,7 @@ nl:
     walletelectrum: "Electrum is gericht op snelheid en eenvoud en is zeer licht voor uw computer. Het laat de ingewikkeldere taken van het Bitcoin-systeem uitvoeren door servers elders en het biedt de mogelijkheid om uw portemonnee terug te vinden met behulp van een geheime zin."
     walletmsigna: "mSIGNA is een geavanceerde maar makkelijk te gebruiken wallet die gekenmerkt wordt door snelheid en eenvoud, schaalbaarheid op ondernemingsniveau, en een sterke beveiliging. Het ondersteunt BIP32, transacties met meerdere handtekeningen, offline opslag, synchronisatie op meerdere apparaten, en versleutelde elektronische en papieren back-ups. "
     walletbitcoinwallet: "Bitcoin Wallet is eenvoudig te gebruiken en betrouwbaar, terwijl het ook veilig en snel is. De visie is decentralisatie en \"zero trust\"; er is geen centrale dienst nodig voor de Bitcoin-gerelateerde handelingen. De applicatie is een goede keuze voor niet-technische personen."
+    walletedgewallet: "Edge is een mobiele Bitcoin-portemonnee met Segwit-ondersteuning die hoge niveaus van privacy, beveiliging en decentralisatie zeer vertrouwd en bruikbaar maakt voor de massa. Edge-portemonnees worden altijd automatisch versleuteld, gebackupt, en functioneren zelfs wanneer de Edge-servers uitvallen."
```

Comment:
Please revise or drop change to this file as per comment in: `_translations/en.yml`